spk/ps3netsrv/src/ps3netsrv-starter.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ -r "${CFG_FILE}" ]; then
+    . "${CFG_FILE}"
+fi
+
+/bin/stdbuf -o L -e L ${BIN_FILE} "${PS3_DIR}" >> ${LOG_FILE} 2>&1 &
```

Comment:
No need to redirection to LOG_FILE as generic script does it already: https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.start-stop-status#L39

Reply:
Ok, done